packages/ui/src/scss/_desktop.scss

```diff
@@ -54,10 +57,9 @@ $font-family: -apple-system,
   --font-title: 600 32px/40px #{$font-family};
   --font-title-medium: 600 28px/36px #{$font-family};
   --font-title-small: 600 24px/32px #{$font-family};
-  --font-body-bold: 500 16px/20px #{$font-family};
-  --font-body: 400 16px/20px #{$font-family};
-  --font-body-small: 500 14px/18px #{$font-family};
-  --font-caption: 500 14px/20px #{$font-family};
+  --font-body-bold: 500 14px/20px #{$font-family};
```

Comment:
Seems we do not have `body-bold` font in our design system.

Reply:
yes, it is mapped with mobile design system for not. not entirely AC design system.